meta-security-smack/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -10,10 +10,11 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 # We use the beginning of PV (unexpanded here to prevent a cyclic dependency
 # during resolution apparently caused by ${SRCPV}) to pick the right set of
 # patches.
-SRC_URI_append_smack = " ${SYSTEMD_SMACK_PATCHES_${@d.getVar('PV', False)[0:3]}}"
+#
+# Patches are optional. Hopefully we won't need any for systemd >= 229.
+SRC_URI_append_smack = " ${@d.getVar('SYSTEMD_SMACK_PATCHES_' + d.getVar('PV', False)[0:3], True) or ''}"
 
 SYSTEMD_SMACK_PATCHES_216 = " \
```

Comment:
Does this mean that the patches are applied only for systemd versions 216, 219, 225, and 228?